android/expoview/src/main/java/host/exp/exponent/ExpoApplication.java

```diff
@@ -55,7 +54,14 @@ public void onCreate() {
     }
 
     if (!Constants.IS_DETACHED) {
```

Comment:
I think the logic here should just be
```
if (!Constants.isStandaloneApp()) {
   KernelConstants.MAIN_ACTIVITY_CLASS = LauncherActivity.class;
}
```
since this field is already set to `MainActivity` by default.

can we get rid of this `IS_DETACHED` constant as well? It looks like this is the only place it's used now.

Reply:
There's one other place `IS_DETACHED` is used:

https://github.com/expo/expo/blob/a3011e90aea5c5020bfe627b582584468168cb87/android/expoview/src/main/java/host/exp/exponent/analytics/Analytics.java#L84

But you're right to assume it isn't **really** used, as `ANALYTICS_ENABLED = !isStandaloneApp()` and if `ANALYTICS_ENABLED == false`, no event is sent to Amplitude. So in fact no matter what the `IS_DETACHED` value is, it won't ever be tracked.